pkgs/by-name/wa/watchyourlan/package.nix

```
@@ -0,0 +1,25 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+buildGoModule rec {
+  pname = "watchyourlan";
+  version = "2.0.0";
+
+  vendorHash = "sha256-+7CMubbCrl+DsGmN9/2jCQ6zLDKvjJ6PdJx8iI1vKOQ=";
+
+  src = fetchFromGitHub {
+    owner = "aceberg";
+    repo = "WatchYourLAN";
+    rev = version;
+    hash = "sha256-Z4UitJKWZXGypNJ737DQNGuyDXIbyT7a1wOSc6N2s1k=";
+
```

Does build, but fails at runtime due to `arp-scan` not being found.

```suggestion

  nativeBuildInputs = [ makeBinaryWrapper ];
  
  postFixup = ''
    wrapProgram $out/bin/WatchYourLAN \
      --prefix PATH : '${lib.makeBinPath [ arp-scan ]}'
  '';
```